analysis/02_pipe.R

```diff
@@ -44,7 +44,7 @@ cohort_end_date <- as.Date("2021-12-14")
 #Used to split time since COVID exposure; when there are time periods with no events then
 #a reduced number of time periods is used (need 197 instead of 196 as time periods are split using [ , ) 
 
-cuts_days_since_expo <- c(14, 28, 56, 84, 197) 
+cuts_days_since_expo <- c(28, 197) 
 cuts_days_since_expo_reduced <- c(28,197) 
```

Comment:
If we change `cuts_days_since_expo` to `c(28, 197)` then `cuts_days_since_expo_reduced` becomes redundant. I think we should keep the 2 sets of time cuts and make sure the reduced version is used any when we have a reduced number of events.